app/src/main/java/com/kickstarter/ui/viewholders/InitialCommentLoadErrorViewHolder.kt

```diff
@@ -0,0 +1,11 @@
+package com.kickstarter.ui.viewholders
```

Comment:
This ViewHolder can be deleted and use instead `EmptyViewHolder` already existing in the app